### DISCUSSION TITLE
Automatically denied 8 times

### DISCUSSION THREAD

**Discussion by bliss02** [community/community#52605] [GitHub Education]
I'm trying to apply for the Student Benefits but it seems to be impossible. The first attempt didn't have my school name on it so on the second attempt I provided a collage with my report card, named planner, GCSE Timetable, Class schedule. All of those were dated and had my name on it apart from the GCSE Timetable which didn't have my name.  

This seems to be the only attempt I've had which was actually looked at by a human because after a day it was denied and I was told I didn't use the exact school name which was fair enough, but after making this change, every single attempt at verifying gets denied as soon as I upload it, and I can only take pictures from my camera I cant choose files, I've tried uploading genuinely all the proof I have but its somehow not enough, these are the things I've tried (I've also put them together in the image not just 1 piece of evidence each time) . Report Card, Class Schedule, Planner, Awards, GCSE Examination Timetable (Named this time).

This time all of those had my name and a date on and were official documents from the school, also since the first attempt I've had a short description and my full name on my GitHub profile, its the exact same name with the exact same capitalisation as all the other attempts.

The pictures are easily readable and by no means blurry, I use the back camera of an almost new iPhone 12 Pro and I have cleaned all of the lenses and even tried using the different lenses on it. If that is not sufficient enough to take the pictures I don't understand what I'm meant to do, the lighting is fine in each image and I've tried moving into different rooms with different lighting, it's daytime right now so there's also plenty of natural lighting and I have allowed github to see my location.

Any help would be appreciated, I really don't know what to do, I have tried all the suggestions noted on other peoples posts, and it seems like the only way I can verify is with an admins help.

![what](https://user-images.githubusercontent.com/129511144/231433062-740fc64c-ff9b-4216-9924-09b9f3dfd7da.JPG)

**Comment by samus-aran** [comment]
Hi @bliss02 👋 

Usually with this type of rejection reason (lack of) we would suggest you re-assess your evidence and acquire or utilise a more preferred form of evidence. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. Ensure you have selected your verified academic email when applying. 

Let us know how you get along 👍
